### PR TITLE
Add PLOS ONE DOI for Cichlid manuscript, remove preprints

### PR DESCRIPTION
**Pull request by habi**

### CODE REVIEW

File: catalog.yml [hunk 1]
```diff
@@ -170,14 +170,13 @@
   preprint_citation: doi:10.22541/au.160979530.02627436/v1
 - repo_url: https://github.com/habi/acinar-analysis-manuscript
   html_url: https://habi.github.io/acinar-analysis-manuscript/
-  preprint_citation: doi:10.1101/786681
   journal_citation: doi:10.1371/journal.pone.0257349
 - repo_url: https://github.com/habi/zmk-tooth-cohort-method-manuscript
   html_url: https://habi.github.io/zmk-tooth-cohort-method-manuscript/
-  preprint_citation: doi:10.1101/2020.10.27.356998
```
[the remark this answers is not in the document]
Comment (habi) [reply catalog.yml:177]: Of course, I just thought that's not necessary.

File: catalog.yml [hunk 2]
```diff
@@ -170,14 +170,13 @@
...
   journal_citation: doi:10.1186/s12903-021-01551-x
 - repo_url: https://github.com/habi/EAWAG-manuscript
   html_url: https://habi.github.io/EAWAG-manuscript/
+  journal_citation: 10.1371/journal.pone.0291003
```
[the remark this answers is not in the document]
Comment (habi) [reply catalog.yml:179]: Thanks a lot

**Comment by habi** [comment]
I've added the two deleted preprints back and added the preprint for the newest manuscript.

**Review by dhimmel — APPROVED** [review]

File: catalog.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (dhimmel) [reply catalog.yml]: Makes sense. The purpose of the catalog is not that well defined, but aspirationally it has some relevance for parties who are interested in tracking the research lifecycle of github manuscript all the way through journal publication.

**Pull request merged**